website/cue/reference/configuration.cue

```diff
@@ -37,7 +37,7 @@ configuration: {
 			}
 		}
 
-		expire_metrics: {
+		expire_metrics_secs: {
```

Comment:
Technically according to https://github.com/vectordotdev/vector/blob/master/docs/DEPRECATION.md#deprecation we should leave the old option too and document it as deprecated.

Reply:
I'd personally be ok with not including it in the docs given the situation, but yeah - letter of the spec we should.